doc/en/components/grids/_shared/row-pinning.md

```diff
@@ -79,7 +77,7 @@ this.grid.getRowByIndex(0).pinned = true;
 ```
 
 ```razor
-this.Grid.PinRowAsync("ALFKI", 0);
+this.Grid.GetRowByIndex(0).Pinned = true;
```

Comment:
I don't think directly setting the property works in Blazor. Hence why the method was used instead.